Tests/CommandsTests/PackageRegistryToolTests.swift

```diff
@@ -492,7 +441,7 @@ final class PackageRegistryToolTests: CommandsTestCase {
 
             // Generate signature
             let signaturePath = temporaryDirectory.appending(component: "signature")
-            try await PackageArchiveSigner.sign(
+            try PackageArchiveSigner.sign(
```

Comment:
This change is included in https://github.com/apple/swift-package-manager/pull/6237 already